.travis.yml

```diff
@@ -47,10 +47,12 @@ jobs:
       - PYTHON=3.7
       - NUMPY=1.16.2
       - PANDAS>=0.24.1
+      - UPSTREAM_SPARSE=1
```

Comment:
Thoughts on consolidating this with the `UPSTREAM_DEV` case below?

Reply:
I purposely didn't add it there, so we can test both master _and_ current release.

Reply:
Sure that makes sense.

What I meant to ask (and failed to), is any thoughts on having the dev version of sparse installed alongside the other dev versions and making this case the stable release of sparse?

Reply:
I think the answer here is that `__array_function__` was very recently added to Sparse by @pentschev and we currently need upstream to test it. Though if it is a concern I can make another release within a few hours.

Reply:
Sure. Am just suggesting we move both of these (`UPSTREAM_SPARSE` and `*array_function`) into the next CI case, which is already using other nightlies (including one for NumPy).

Not pushing on releasing things or other complex changes. Just encouraging all the tests that require unreleased changes to live in the same test case.

Does that make sense? Please feel free to ask questions if not.

Reply:
I understand your point @jakirkham, but I'd like these tests to be executed also during PRs, which is not the case for `UPSTREAM_DEV=1` case. I've been working on these issues very frequently, having theses tests outside of PRs will incur in missing such errors. I also think we should test `__array_function__` against NumPy 1.16.2, it should work there too, not just with upstream NumPy.

My idea is that we should do that later, once we're more confident with the stability of these new Dask, NumPy and Sparse features, but for now it's best to have more situations tested during PRs.